doc/manuals/devel/mongoBackend.md

```diff
@@ -28,556 +28,562 @@
+* `mongoAttributesForEntityType()` is invoked from a service routine (step 1). This can be from either `getEntityType()` (which resides in `lib/serviceRoutinesV2/getEntityType.cpp`) or `getAttributesForEntityType()` (which resides in `lib/serviceRoutines/getAttributesForEntityType.cpp`).
+* Depending on `-reqMutexPolicy`, the request semaphore may be taken (read mode) (step 2). See [this document for details](semaphores.md#mongo-request-semaphore). 
+* A list of entity attributes corresponding to the entity type is retrieved from the database, using `runCollectionCommand()` in the `connectionOperations` module to run an aggregation command (steps 3 and 4).
 * If attribute detail is enabled (i.e. `noAttrDetail` set to `false`) a loop iterates on every attribute in order to:
-   * Invoke `getAttributeTypes()` to get the different types of the attribute (along the entities of the same entity type) (step 5).
-   * The information is retrieved from DB using `collectionQuery()` function in the `connectionsOperation` module (steps 6 and 7).
-* If request semaphore was taken in step 2, then it is released before returning to calling function (step 8).
+   * Invoke `getAttributeTypes()` to get the different types of the attributes (along with the entities of the same entity type) (step 5).
+   * The information is retrieved from the database using `collectionQuery()` in the `connectionsOperation` module (steps 6 and 7).
+* If the request semaphore was taken in step 2, then it is released before returning (step 8).
 
 These functions use `EntityTypeVectorResponse` (two first cases) and `EntityTypeResponse` objects in order to return results to calling service routine.
 
-Note the usage of `noAttrDetails` parameter in `mongoEntityTypes()` and `mongoAttributesForEntityType()` in order to avoid a (potentially costly) process to get types of the attributes associated to an entity type (implemented by the `getAttributeTypes()` function).
+Note the usage of the `noAttrDetails` parameter in `mongoEntityTypes()` and `mongoAttributesForEntityType()` in order to avoid a (potentially costly) process to get types of the attributes associated to an entity type (implemented by `getAttributeTypes()`).
 
-All the above functions heavily relies in the MongoDB aggregation framework. You should be familiar with this framework (and with the `entities` collection structure, [described as part of the database model in the administration documentation](../admin/database_model.md#entities-collection)) in order to understand how they work.
+All the above functions heavily rely on the MongoDB aggregation framework. You should be familiar with this framework (and with the `entities` collection structure, [described as part of the database model in the administration documentation](../admin/database_model.md#entities-collection)) in order to understand how the functions work.
 
 [Top](#top)
 
 #### `mongoCreateSubscription` (SR2)
 
-Encapsulates the contex
```

Comment:
Here is another unfinished phrase.

Original: Among them (details in source code) we highly `setNotification()` (step 5), due to ...


We highly ... what?